docs/platforms/full-dotnet/existing-db.rst

```diff
@@ -58,7 +58,10 @@ Now it's time to create the EF model based on your existing database.
 * :menuselection:`Tools –> NuGet Package Manager –> Package Manager Console`
 * Run the following command to create a model from the existing database
 
-.. literalinclude:: _static/reverse-engineer-command.txt
+.. code-block:: text
+
+    Scaffold-DbContext "Server=(localdb)\mssqllocaldb;Database=Blogging;Trusted_Connection=True;" Microsoft.EntityFrameworkCore.SqlServer
+
```

Comment:
Correct, you don't really *need* it in the other place, it's just idiomatic of ASP.NET applications to put entity classes in the Models folder.